### DISCUSSION TITLE
Word "id" represented as two tokens

### DISCUSSION THREAD

**Discussion by sillentkill21** [explosion/spaCy#11154] [Help: Coding & Implementations] [answered]
## How to reproduce the behaviour
`import spacy

nlp = spacy.load('en_core_web_sm')

doc = nlp("id of something")

for token in doc:
    print(token.text, token.pos_)`

## Your Environment

* Operating System: Windows 10 64bit/Ubuntu 20.04
* Python Version Used: 3.8
* spaCy Version Used: 3.2.3
* Language models: en_core_web_sm, en_core_web_md

## Additional info
After testing the word "id" in multiple situations, the results were the same. Word "id" was represented as two tokens "i" and "d". Is this the expected behavior?

![image](https://user-images.githubusercontent.com/56270655/179543828-a562fc92-46a9-4c33-8e0e-c04818fe8d3c.png)

**Comment by polm** [comment, marked as answer]
This is intended behavior, see #10455. The English tokenizer treats "id" as a typo for "I'd" by default, though you can change it.
